--- README.latex	2005-01-31 09:09:57.000000000 +0100
+++ README.latex.luis	2005-01-31 09:07:07.000000000 +0100
@@ -1,5 +1,5 @@
-\documentclass[10pt,a4paper,french]{article}
-\usepackage{babel}
+\documentclass[10pt,a4paper]{article}
+\usepackage[english,french]{babel}
 \usepackage{ae}
 \usepackage{aeguill}
 \usepackage{shortvrb}
@@ -84,7 +84,7 @@
 \hypertarget{documentation-d-administration-du-produit-scrib}{}
 \section{Documentation d'administration du produit Scrib}
 \tableofcontents
-
+\newpage
 \bigskip
 
 
